CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+Contributor License
+-------------------
+
+VOLTTRON is an Eclipse Foundation project.  If this is your first time contributing to an Eclipse Foundation project, you'll need to sign the [Eclipse Contributor Agreement][ECA].
+
+- [Create an account](https://dev.eclipse.org/site_login/createaccount.php) on dev.eclipse.org
+- Open your [Account Settings tab](https://dev.eclipse.org/site_login/myaccount.php#open_tab_accountsettings), enter your GitHub ID and click Update Account
+- Read and [sign the ECA](https://dev.eclipse.org/site_login/myaccount.php#open_tab_cla)
+- Use the exact same email address for your Eclipse account and your commit author.

```

Comment:
This link doesn't work for me either. https://accounts.eclipse.org/user/609877/eca/3.1.0 works

Reply:
once you create an account and sign in it will work....I am not going to embed the userid into the url.

Reply:
@schandrika  is right...therefore added an image on how to do this.